src/gui/tray/ActivityItemContent.qml

```diff
@@ -81,18 +82,20 @@ RowLayout {
             readonly property int negativeLeftMargin: -((width / 2) +
                                                         ((width - paintedWidth) / 2) +
                                                         ((thumbnailImageLoader.width - thumbnailItem.imageWidth) / 2) +
-                                                        (thumbnailImageLoader.width - thumbnailImageLoader.item.paintedWidth) / 2)
+                                                        ((thumbnailImageLoader.width - thumbnailImageLoader.item.paintedWidth) / 2) +
+                                                        (thumbnailItem.thumbnailRadius / 4))
             readonly property int negativeTopMargin: -((height / 2) +
                                                        ((height - paintedHeight) / 2) +
                                                        ((thumbnailImageLoader.height - thumbnailItem.imageHeight) / 4) +
-                                                       ((thumbnailImageLoader.height - thumbnailImageLoader.item.paintedHeight) / 4))
+                                                       ((thumbnailImageLoader.height - thumbnailImageLoader.item.paintedHeight) / 4) +
+                                                       (thumbnailItem.thumbnailRadius / 4))
             anchors.verticalCenter: if(model.thumbnail === undefined) parent.verticalCenter
             anchors.left: model.thumbnail === undefined ? parent.left : thumbnailImageLoader.right
             anchors.leftMargin: if(model.thumbnail !== undefined) negativeLeftMargin
             anchors.top: if(model.thumbnail !== undefined) thumbnailImageLoader.bottom
             anchors.topMargin: if(model.thumbnail !== undefined) negativeTopMargin
 
-            cache: true
+            cache: false
```

Comment:
are you sure it is worth removing it from cache ?

Reply:
Nope, was just testing. Reverted to cache being true